### PR TITLE
Feature iview job post

### PR DESCRIPTION
**Pull request by Bhavisha-iView**

### CODE REVIEW

**Comment by PareshMayani** [comment]
@Bhavisha-iView thank you for sending a PR. Could you please either Rebase or send a fresh PR with only your commits?

**Comment by PareshMayani** [comment]
> @Bhavisha-iView thank you for sending a PR. Could you please either Rebase or send a fresh PR with only your commits? Also, you need to send a PR to `develop` branch

@Bhavisha-iView I've changed the Base to `develop` branch. I would request you to verify your PR changes once and confirm.

**Comment by Bhavisha-iView** [comment]
> > @Bhavisha-iView thank you for sending a PR. Could you please either Rebase or send a fresh PR with only your commits? Also, you need to send a PR to `develop` branch
> 
> @Bhavisha-iView I've changed the Base to `develop` branch. I would request you to verify your PR changes once and confirm.

@PareshMayani I have verified changes, thanks.

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -674,6 +674,64 @@
           "experience": "1 to 2 Years experience"
         }
       ]
+    },
+    {
+      "id": 16,
+      "companyName": "iView Labs Pvt. Ltd",
+      "location": "Ahmedabad, India",
+      "bio": "iView Labs is a leading cloud-based application development company that provides software solutions to US, Europe, and Australian businesses.",
+      "companyLogoUrl": "https://drive.google.com/file/d/1uMPCQCefDW97WQa22a0Gtj03lZrOqLL1/view?usp=share_link",
```
Comment (JaldeepAsodariya) [inline public/data/job-openings-data.json:683]: This URL will not work (Company logo will not load), Please refer other jobs object.

**Review by JaldeepAsodariya — CHANGES_REQUESTED** [review]

**Comment by JaldeepAsodariya** [comment]
@Bhavisha-iView Can you please check the requested changes?
Thanks

**Pull request merged**